qemu/tests/cfg/snp_attestation.cfg

```diff
@@ -0,0 +1,45 @@
+- snp_attestation:
+    type = snp_basic_config
+    only Linux
+    only HostCpuVendor.amd
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    image_snapshot = yes
+    mem = 8192
+    smp = 8
+    required_qemu = [9.1.0, )
+    vm_secure_guest_type = snp
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
```

Comment:
Recently, VT merged https://github.com/avocado-framework/avocado-vt/pull/4276 so now supports dynamic value, but in qemu commit, still suggest set reduced-phys-bits always to 1, so you can consider remove `vm_sev_cbitpos`?

But you from AMD, maybe you know more than me, so if I am wrong, you can correct me. 😄